.github/workflows/ci.yml

```diff
@@ -93,7 +93,12 @@ jobs:
     - name: Unit tests
       run: |
         source .env/bin/activate
-        touch .test_durations
+        if [[ -f .test_durations ]]
+        then
+          touch .test_durations
```

Comment:
if the file already exists touch does nothing; so let's just do the echo only if the file doesn't exist